docs/topics/rn-known-issues.adoc

```diff
@@ -8,7 +8,7 @@
 
 At the time of release, the following known issues have been identified as major issues worth highlighting.
 
-For a complete list of all known issues, see the list of link:https://issues.redhat.com/issues/?filter=12385749[{ProductVersion} known issues] in Jira.
+For a complete list of all known issues, see the list of lihttps://issues.redhat.com/issues/?filter=12393203[MTA {ProductVersion} known issues] in Jira.
```

Comment:
lihttps://issues.redhat.com/issues/?filter=12393203

I think it should be link https .

Reply:
Fixed.